src/applications/edu-benefits/0994/content/benefitsEligibility.jsx

```diff
@@ -4,7 +4,7 @@ export const eligibilityDescription =
   'Have you applied for VA education benefits in the past?';
 
 export const eligibilityInstructions = (
-  <div className="feature">
+  <va-summary-box class="vads-u-margin-y--3">
```

Comment:
I wasn't sure how to run this app so that I could preview. Can the edu-benefits code owners assist?

Reply:
yarn watch --env api=https://staging-api.va.gov/ entry=0994-edu-benefits

Reply:
This command should run the 0994 form

Reply:
/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-099

Reply:
I'm having some issues with the VM on my GFE so I'll have to restart my machine